pkg/tests/handlers/accounts.go

```diff
@@ -125,3 +133,78 @@ func TestQuotaCost(options *helpers.TestOptions) error {
 	return TestStaticEndpoint(options)
 
 }
+
+// Test Cluster Authorizations
+func TestClusterAuthorizations(options *helpers.TestOptions) error {
+	testName := options.TestName
+	// Vegeta Results File
+	fileName := fmt.Sprintf("%s_%s.json", options.ID, testName)
+	resultFile, err := helpers.CreateFile(fileName, options.OutputDirectory)
+	if err != nil {
+		return err
+	}
+	defer resultFile.Close()
+
+	targeter := func(t *vegeta.Target) error {
+
+		// Each Cluster uses a UUID to ensure uniqueness
+		clusterId := uuid.NewV4().String()
+		t.Method = http.MethodPost
+		t.URL = options.Path
+		t.Body = clusterAuthorizationsBody(clusterId)
+
+		return nil
+	}
+
+	// Create a Metrics bucket for this test run
+	options.Metrics[testName] = new(vegeta.Metrics)
+	defer options.Metrics[testName].Close()
+
+	// Execute the HTTP Requests; repeating as needed to meet the specified duration
+	for res := range options.Attacker.Attack(targeter, options.Rate, options.Duration, options.TestName) {
+		result.Write(res, resultFile)
+		options.Metrics[testName].Add(res)
+	}
+
+	log.Printf("Results written to: %s", fileName)
+
+	if options.WriteReport {
+		err = report.Write(fmt.Sprintf("%s_%s-report", options.ID, options.TestName), options.OutputDirectory, options.Metrics[testName])
+		if err != nil {
+			return err
+		}
+	}
+
+	return nil
+}
+
+func clusterAuthorizationsBody(clusterID string) []byte {
+	buff := &bytes.Buffer{}
+	reservedResource := v1.NewReservedResource().
+		ResourceName("m5.xlarge").
+		ResourceType("compute.node.aws").
+		Count(4).
+		BillingModel("standard")
+
+	clusterAuthReq, err := v1.NewClusterAuthorizationRequest().
+		ClusterID(clusterID).
+		ProductID("osd").
+		CloudProviderID("aws").
+		AccountUsername(helpers.AccountUsername).
+		Managed(true).
+		Reserve(true).
+		BYOC(true).
+		AvailabilityZone("single").
+		Resources(reservedResource).
+		Build()
```

Comment:
@chentex Sorry for nitpicking do you mind making all of these strings into `consts` somewhere?